src/Compilers/Core/Portable/Symbols/ITupleTypeSymbol.cs

```diff
@@ -14,7 +13,7 @@ namespace Microsoft.CodeAnalysis
     /// </remarks>
     public interface ITupleTypeSymbol : ITypeSymbol
```

Comment:
NamedType is cleaner on the IDE for some things.

For example, we can then appropriately do cross language work more easily.  For example, if i generate method from C# to VB and i pass a tuple, then i can generate the right VBism properly.  And similarly, if i pass a wellformed tuple from VB to C# then that should also just work.  It makes a bit wonkier if the same metadata type appears to be different symbol types in VB and C#.